examples/extensions/deployers/sources/custom_deployer_sources.rst

```diff
@@ -0,0 +1,76 @@
+.. examples_extensions_deployers_sources:
+
+Copy sources from all your dependencies
+=======================================
+
+
+
+Please, first of all, clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/examples/extensions/deployers/sources
```

Comment:
I get:
```
cd: no such file or directory: examples2/examples/extensions/deployers/sources
```
is it expected? or typo in path?

Reply:
Yes, this is expected as the PR in examples2 has not yet been merged